spg/My-List-Util/lib/My/List/Util.pm

```diff
@@ -1,161 +1,164 @@
-package My::List::Util;

```

Comment:
I was getting some windows line endings showing up in my diffs, so I tried to fix it. That just made  caused the whole thing to show up as a diff for some reason.